bin/setup_configuration.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# setup initial configuration using environment variables
+# Run this script from the root of the repository
+
+#set -e
+${SCRIPTPATH}/wait_for_db.sh
```

Comment:
Dit gaat niet werken zonder wait_for_db.sh script: https://github.com/open-zaak/open-zaak/blob/main/bin/wait_for_db.sh

Ik ken setup_configuration niet goed genoeg om te weten of/hoe deze script automatisch wordt uitgevoerd, in Open Zaak zie ik het niet terug. Is dit iets wat je in je helm chart doet?

Reply:
Hallo @alextreme ,

Het wait_for_db.sh script is nu ook toegevoegd.

Het script is door Anne gemaakt en opgeleverd en het is inderdaad makkelijk om deze te hebben om aan te roepen vanuit een init container, zie: https://github.com/open-zaak/open-zaak/blob/9ac277e8ed012521b2c1c709b4f683d45d90a37c/docker-compose.yml#L68 en https://github.com/maykinmedia/charts/blob/51c6860f1943d6cf03e80c69dc71ba41dadbac7e/charts/openzaak/templates/deployment.yaml#L51

Het setup_configuration python commando aanroepen zou ik ook zonder script kunnen doen, wat als grootste voordeel heeft dat ik niet hoef te wachten op releases.